google/cloud/apigeeregistry/v1/consistency_report.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Holds words labeled into the same group
+
+syntax = "proto3";
+
+package google.cloud.apigeeregistry.v1;
+
+import "google/api/field_behavior.proto";
+import "google/cloud/apigeeregistry/v1/word_group.proto";
+
+message ConsistencyReport {
+    string id = 1 [ (google.api.field_behavior) = REQUIRED ];
+    string  kind = 2 [ (google.api.field_behavior) = REQUIRED ]; 
+
+   message Variation {
+        string term = 1 [ (google.api.field_behavior) = REQUIRED ];
+        WordGroup cluster = 2 [ (google.api.field_behavior) = REQUIRED ]; 
+   }
+   // Stores the variations detected in the current upload
+   repeated Variation current_variations = 3;
+
+   // Stores a snapshot of the variations detected in the last upload. 
```

Comment:
Let's try to remove the word "upload" because it suggests a dependency on a sequence of events that isn't necessarily relevant. What exactly are we representing? If we can describe that more clearly, that would be a good replacement for "upload". I think this might be something like "the variations of a spec with respect to a collection of specs".